test-harness/runner.cfm

```diff
@@ -17,6 +17,8 @@
 <cfparam name="url.coverageBlacklist"				default="/testbox,/coldbox,/tests,/modules,Application.cfc,/index.cfm">
 <!---<cfparam name="url.coverageBrowserOutputDir"		default="#expandPath( '/tests/results/coverageReport' )#">--->
 <!---<cfparam name="url.coverageSonarQubeXMLOutputPath"	default="#expandPath( '/tests/results/SonarQubeCoverage.xml' )#">--->
+<!--- Enable batched code coverage reporter, useful for large test bundles which require spreading over multiple testbox run commands. --->
+<!--- <cfparam name="url.isBatched"						default="false"> --->
```

Comment:
Remove commented code above

Reply:
I hear you, but I feel like this needs to remain here for documentation reasons. Isn't this directory what `box testbox generate harness` uses to create a test harness?

Reply:
I would prefer that stays-- those comments have served as documentation for the code coverage features people can opt into.  We'll need to ensure we have additions to the TestBox docs as well explaining how to use this feature.

Reply:
I'm fine with that.

Reply:
> We'll need to ensure we have additions to the TestBox docs as well explaining how to use this feature.

Noted - some of the existing options are missing from the docs AND this file as well. I'm going to go ahead and add those.